packages/teams-js/src/internal/handlers.ts

```diff
@@ -177,7 +175,7 @@ export function handleThemeChange(theme: string): void {
     HandlersPrivate.themeChangeHandler(theme);
   }
 
-  if (Communication.childWindow) {
+  if (Communication.childWindow && (GlobalVars.allowMessageProxy || GlobalVars.webAuthWindowOpen)) {
```

Comment:
This and the load handler below have all been here for a long time, so I decided for now to gate them behind the `allowMessageProxy` logic instead of outright removing them.